ProjectLighthouse.Servers.GameServer/Controllers/Login/LoginController.cs

```diff
@@ -57,27 +49,27 @@ public async Task<IActionResult> Login()
 
         string ipAddress = remoteIpAddress.ToString();
 
-        string? username = npTicket.Username;
+        string username = npTicket.Username;
 
-        if (username == null)
+        if (String.IsNullOrEmpty(username))
         {
             Logger.Warn("Unable to determine username, rejecting login", LogArea.Login);
             return this.Forbid();
         }
 
-        await this.database.RemoveExpiredTokens();
+        await database.RemoveExpiredTokens();
```

Comment:
Doesn't using the primary constructor DbContext variable cause a warning? I forget how it works exactly in LH but in my own personal projects it causes a warning.